lib/index.ts

```diff
@@ -191,7 +203,7 @@ export async function validate(
  * @param size component set package size in bytes
  * @param errorReporter called when there is a validation error
  */
-export function validatePackageSize(size: number, errorReporter: (errorMessage: string) => void) {
+export function validatePackageSize(size: number, errorReporter: (errorMessage: string) => void): boolean {
```

Comment:
What's the eslint rule for the return types? I don't mind them being required for public APIs, but might be annoying if they are also required for private functions.

Reply:
In this case we use the default settings: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md

If we want to tweak this we probably also want this for the other repos.